man/systemd.resource-control.xml

```diff
@@ -198,6 +198,19 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>CPUIdle=</varname></term>
+
+        <listitem>
+          <para>Takes a boolean argument, defaults to false.
+          When true, mark the cgroup for "idle scheduling", which means that it will get CPU resources only
+          when there are no processes not marked in this way to execute in this cgroup or its siblings.
+          Note that setting CPUWeight is illegal if CPUIdle is true.
+          This setting corresponds to the cpu.idle cgroup attribute.
+          </para>
+        </listitem>
+      </varlistentry>
```

Comment:
also needs an update to `CPUWeight=` docs, saying that `CPUWeight=` has no effect if `CPUIdle=` is set (that's how this works, no?)

Also would be good to briefly say what happens if i have a slice with idle mode and a service below it which does not have idle mode. Is the service then of idle mode anyway?

Also would be good to briefly mention how the process SCHED_IDLE stuff relates to this per cgroup stuff: what wins?